src/SFA.DAS.EmployerIncentives.Web/Controllers/PaymentsController.cs

```diff
@@ -69,7 +69,7 @@ public async Task<IActionResult> ListPaymentsForLegalEntity(string accountId, st
             {
                 Applications = submittedApplications,
                 SortField = sortField,
-                BankDetailsStatus = getApplicationsResponse.BankDetailsStatus,
+                ShowBankDetailsCalltoAction = getApplicationsResponse.BankDetailsStatus == BankDetailsStatus.NotSupplied || getApplicationsResponse.BankDetailsStatus == BankDetailsStatus.Rejected,
```

Comment:
The bank details content panel isn't just shown for these statuses. See the EI-390 branch

Reply:
In EI-390 it' shown for BankDetailsStatus.NotSupplied which is included in the code above.